Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -519,7 +519,10 @@ void BlockFormattingContext::layout_inline_children(BlockContainer const& block_
         // NOTE: min-width or max-width for boxes with inline children can only be applied after inside layout
         //       is done and width of box content is known
         auto used_width_px = context.automatic_content_width();
-        auto available_width = AvailableSize::make_definite(used_width_px);
+        // https://www.w3.org/TR/css-sizing-3/#sizing-values
+        // Percentages are resolved against the width/height, as appropriate, of the box’s containing block.
+        auto containing_block_width = m_state.get(*block_container.containing_block()).content_width();
```

Comment:
it does not work for grids because containing block of grid is not grid container but grid area formed by tracks it belong to.

Reply:
Hmm, that is what I brought up in the PR description, but I had thought that perhaps correctly pointing `containing_block` to the grid area would be a separate issue. Or is there already wiring/a function I can call to retrieve the correct containing block in a grid context?